src/backend/distributed/utils/citus_outfuncs.c

```diff
@@ -469,6 +469,12 @@ OutRelationRowLock(OUTFUNC_ARGS)
 	WRITE_ENUM_FIELD(rowLockStrength, LockClauseStrength);
 }
 
+void OutTaskQuery(OUTFUNC_ARGS) {
+	WRITE_LOCALS(TaskQuery);
+	WRITE_NODE_TYPE("TASKQUERY");
+
+	WRITE_ENUM_FIELD(queryType, TaskQueryType);
```

Comment:
We should output taskQuery->data here with a switch statement I guess as it is a union ?

Reply:
Yes

Reply:
Okay updated.